joomla/joomla-cms
# [4.0] field simple color fix

Pull Request for Issue # .

### Summary of Changes
- Revert the removal of the rgb to hex function

### Testing Instructions
Apply patch and check that the selected value is retained after save (template autum has a field for testing) 


### Expected result



### Actual result



### Documentation Changes Required
